cranelift/src/utils.rs

```diff
@@ -46,12 +46,21 @@ pub fn parse_sets_and_triple(
     flag_triple: &str,
 ) -> Result<OwnedFlagsOrIsa, String> {
     let mut flag_builder = settings::builder();
-    parse_options(
+
+    // Collect unknown system-wide settings, so we can try to parse them as target specific
```

Comment:
There is already a way to have target specific settings: `target riscv32 supports_m=1`

https://github.com/bytecodealliance/wasmtime/blob/d6b158992697546dfda982495bfa7572668a9888/cranelift/filetests/filetests/isa/riscv/expand-i32.clif#L4

Reply:
The function I'm modifying is called by the CLI flags parser, though; is there a way to pass target-specific flags through the CLI that I missed?

Reply:
I'll assume no, then :)